tfiac.py

```diff
@@ -62,7 +62,6 @@ def __init__(self, host):
         self._status = {}
         self._name = None
         self._available = True
-        self.update()
```

Comment:
Is it not needed to call 'update' at the beginning anymore?

Reply:
Now you get the following error:

> Traceback (most recent call last):
>   File "/Users/pavlom/Projects/home-assistant/homeassistant/helpers/entity_platform.py", line 352, in _async_add_entity
>     await entity.async_update_ha_state()
>   File "/Users/pavlom/Projects/home-assistant/homeassistant/helpers/entity.py", line 232, in async_update_ha_state
>     state = self.state
>   File "/Users/pavlom/Projects/home-assistant/homeassistant/components/climate/__init__.py", line 205, in state
>     if self.is_on is False:
>   File "/Users/pavlom/Projects/home-assistant/homeassistant/components/climate/tfiac/__init__.py", line 116, in is_on
>     self._client.status[ON_MODE])
> KeyError: 'is_on'

Reply:
Yes and we are: https://github.com/fredrike/home-assistant/blob/ab864c0d84427ce38f95b73bf97671e4c2b9ac90/homeassistant/components/climate/tfiac.py#L61

We are however (afaik) not allowed to do async tasks in init.